routes/reminder.js

```diff
@@ -24,23 +25,18 @@ router
   .post(utils.validateUserId, async (req, res) => {
     const reminder = req.body;
     let userId = req.params.userId;
-    let title = reminder.title;
-    let textBody = reminder.textBody;
-    let priority = Number.parseInt(reminder.priority);
-    let tag = reminder.tag;
-    let repeating =
-      reminder.repeating === "true" ||
-      reminder.repeating === true ||
-      !(reminder.repeating === "false")
-        ? true
-        : false;
```

Comment:
this logic needs to be present and xss check needs to be done before this logic

do not remove any existing logic from the code